server/channels/app/channel_bookmark_test.go

```diff
@@ -125,13 +129,18 @@ func TestUpdateBookmark(t *testing.T) {
 			HasPreviewImage: true,
 		}
 
-		th.App.Srv().Store().FileInfo().Save(th.Context, file2)
-		th.App.Srv().Store().FileInfo().AttachToPost(th.Context, file2.Id, model.NewId(), th.BasicChannel.Id, model.BookmarkFileOwner)
-		defer th.App.Srv().Store().FileInfo().PermanentDelete(th.Context, file2.Id)
+		_, err = th.App.Srv().Store().FileInfo().Save(th.Context, file2)
+		assert.NoError(t, err)
+		err = th.App.Srv().Store().FileInfo().AttachToPost(th.Context, file2.Id, model.NewId(), th.BasicChannel.Id, model.BookmarkFileOwner)
+		assert.NoError(t, err)
+		defer func() {
+			appErr := th.App.Srv().Store().FileInfo().PermanentDelete(th.Context, file2.Id)
+			assert.Nil(t, appErr)
```

Comment:
It is getting better! Now this also needs to become an `err` (and `require.NoError` should be used)